src/main/java/frc/robot/commands/JoystickTarget.java

```diff
@@ -49,23 +54,15 @@ public void initialize() {
   public void execute() {
     updateConstants();
     double x = m_limelight.getX();
-    double xVelocity = thetaController.calculate(Units.degreesToRadians(x));
-    double reverseFactor = -1.0;
+    double thetaVelocity = thetaController.calculate(Units.degreesToRadians(x)) * targetVelocityCoefficient.get();
     m_drive.drive(
-      reverseFactor * joystickTransform(m_driverController.getRawAxis(OIConstants.kLeftJoyYAxis)) * OIConstants.kMaxMetersPerSec,
-      reverseFactor * joystickTransform(m_driverController.getRawAxis(OIConstants.kLeftJoyXAxis)) * OIConstants.kMaxMetersPerSec,
-      -joystickTransform(m_driverController.getRawAxis(OIConstants.kRightJoyXAxis)) + xVelocity,
+      m_xVelocitySupplier.get(),
+      m_yVelocitySupplier.get(),
+      thetaVelocity,
```

Comment:
Looks like you implemented something close here to what I was suggesting in the context of an earlier commit (switch over to fully automatic rotation control).